pages/_error.js

```diff
@@ -0,0 +1,69 @@
+import NextErrorComponent from 'next/error';
+import PropTypes from 'prop-types';
+
+import * as Sentry from '@sentry/nextjs';
+
+const MyError = ({ statusCode, hasGetInitialPropsRun, err }) => {
```

Comment:
`MyError` maybe we can change this to ErrorPage or something similar?

Reply:
Changed to ErrorPage